web/src/compose_actions.ts

```diff
@@ -373,6 +373,7 @@ export function cancel(): void {
     $("#compose_close").hide();
     clear_box();
     compose_banner.clear_message_sent_banners();
+    compose_banner.clear_messages_fading_banner();
```

Comment:
Do we also want to clear this banner when changing the compose recipient? Otherwise, the displayed recipient can become stale.

(I could also imagine a world where we reword it to avoid displaying the recipient, which can be quite long -- @alya thoughts?)

Reply:
We already addressed this: https://github.com/zulip/zulip/pull/29634#issuecomment-2048284693 ?

Note: The recipient are updated in the banner only when user clicks in compose message textarea. (Same rule as we do with other cases).